tests/jax/systems/mappo_system_test.py

```diff
@@ -1,30 +1,161 @@
 import functools
+from datetime import datetime
+from typing import Any, List
 
-import acme
+import launchpad as lp
 import optax
 import pytest
+from launchpad.launch.test_multi_threading import (
+    address_builder as test_address_builder,
+)
 
+from mava.components.jax.building.distributor import Distributor, DistributorConfig
+from mava.core_jax import SystemBuilder
 from mava.systems.jax import mappo
+from mava.systems.jax.launcher import Launcher, NodeType
 from mava.systems.jax.system import System
+from mava.utils import lp_utils
 from mava.utils.environments import debugging_utils
 from mava.utils.loggers import logger_utils
 
 #########################################################################
 # Full system integration test.
 
 
+class MockLauncher(Launcher):
+    def __init__(
+        self,
+        multi_process: bool,
+        nodes_on_gpu: List = [],
+        sp_trainer_period: int = 10,
+        sp_evaluator_period: int = 10,
+        name: str = "System",
+        terminal: str = "current_terminal",
+    ) -> None:
+        """Create mock launcer component in order to be able to \
+            overwrite the launch method which enables for testing \
+                a full distribured Mava system."""
```

Comment:
typo

Reply:
thanks for catching all of these :sweat_smile: 
sorrted them out.